src/services/VerifiableCredentialService.ts

```diff
@@ -46,32 +46,35 @@ export class VerifiableCredentialService {
   	return VerifiableCredentialService.instance;
   }
 
-  async generateSignedVerifiableCredentialJwt(
-  	sessionItem: ISessionItem | undefined,
-  	credentialSubject: VerifiedCredentialSubject,
-  	evidence: VerifiedCredentialEvidence,
-  	getNow: () => number,
-  ): Promise<string> {
-  	const now = getNow();
-  	const subject = sessionItem?.subject as string;
-  	const verifiedCredential: VerifiedCredential = this.buildVerifiableCredential(credentialSubject, evidence);
-  	const result = {
-  		sub: `${subject}`,
-  		nbf: now,
-  		iss: this.issuer,
-  		iat: now,
-  		vc: verifiedCredential,
-  	};
+	async signGeneratedVerifiableCredentialJwt(result: any): Promise<string> {
```

Comment:
with the use of `any` if we're not able to tighten this type up now due to time constraints please can we put a ticket in the backlog to do this - thanks.

Reply:
removed `any`